elastalert/logsearch-platform-elastalert.yml

```diff
@@ -61,28 +61,19 @@ instance_groups:
               index: logs-platform-*
               query_key:
               - "@source.host"
-              - "event_type"
-              - "signature_name"
-              - "file_path"
               realert:
                 hours: 1
-              alert_subject: "ClamAV alert on host {0}. sig:{1}; path:{2}"
+              alert_subject: "ClamAV alert on host {0}."
               alert_subject_args:
               - "@source.host"
-              - "signature_name"
-              - "file_path"
               alert_text: "{0}"
               alert_text_args:
               - "@message"
-              pagerduty_incident_key: "clamav-alert-{0}-{1}-{2}"
+              pagerduty_incident_key: "clamav-alert-{0}"
               pagerduty_incident_key_args:
               - "@source.host"
-              - "signature_name"
-              - "file_path"
               filter:
-              - bool:
-                  must:
-                  - term:
-                      "@source.component": "clamd"
-                  - match:
-                      "event_type": "ScanOnAccess"
+              - query:
+                  query_string:
+                    query: '@source.component:clamd AND ScanOnAccess AND FOUND'
```

Comment:
Maybe try double quotes.  Might not make a difference.

Reply:
I have tried both, actually.  I settled on this because this worked for me on login.gov:  https://github.com/18F/identity-devops/blob/master/kitchen/cookbooks/identity-elk/templates/default/elastalert_newsudo.yaml.erb

Reply:
We agreed in slack that we'd fix the grok. Let's do that.

Reply:
I don't recall agreeing, but I'll be happy to do that.  In the meantime, I think that it would be a good thing to get clamav alerts going again, which this patch would do.  This patch actually makes this service more resistant to failure if those indexes stop happening again, and we still have all the data in the alert that we did before.  It's just in the alert text rather than the subject.